themes/thecyberden.omp.json

```diff
@@ -0,0 +1,128 @@
+                    "leading_diamond": "",
+                    "trailing_diamond": "\uE0B0",
+                    "properties": {
+                        "prefix": " \uE5FF ",
+                        "style": "full",
+                        "enable_hyperlink": true
+                    }
+                },
+                {
+                    "type": "git",
+                    "style": "powerline",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#193549",
+                    "background": "#c19c00",
+                    "leading_diamond": "",
+                    "trailing_diamond": "",
+                    "properties": {
+                        "display_status": true,
+                        "display_stash_count": true,
+                        "display_upstream_icon": true
+                    }
+                },
+                {
+                    "type": "exit",
+                    "style": "diamond",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#ffffff",
+                    "background": "#2e9599",
+                    "leading_diamond": "",
+                    "trailing_diamond": "\uE0B4",
+                    "properties": {
+                        "always_enabled": true,
+                        "color_background": true,
+                        "display_exit_code": false,
+                        "success_icon": "\uF00C",
+                        "error_icon": "\uF00D",
+                        "error_color": "#f1184c",
+                        "prefix": " "
+                    }
+                }
+            ]
+        },
+        {
+            "type": "prompt",
+            "alignment": "right",
+            "horizontal_offset": 0,
+            "vertical_offset": 0,
+            "segments": [
+                {
+                    "type": "time",
+                    "style": "diamond",
+                    "foreground": "#ffffff",
+                    "background": "#2e9599",
+                    "leading_diamond": "\uE0B6",
+                    "trailing_diamond": "\uE0B4",
+                    "properties": {
+                        "time_format": "15:04:05"
+                    }
+                }
+            ]
+        },
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "newline": true,
+            "segments": [
+                {
+                    "type": "root",
+                    "style": "plain",
+                    "foreground": "#ff0000",
+                    "properties": {
+                        "prefix": "",
+                        "root_icon": "\uE00A"
+                    }
+                },
+                {
+                    "type": "shell",
+                    "style": "powerline",
+                    "powerline_symbol": "",
```

Comment:
this should be `plain` due to the absence of the `powerline_symbol`

Reply:
Changed:
                    "style": "plain",